content/fast/preload-critical-assets/codelab-preload-critical-assets.md

```diff
@@ -132,26 +132,6 @@ unnecessary preload link for this page.
 &lt;/head&gt;
 </pre>
 
-Stylesheet files aren't the only type of asset that can be preloaded. In this
```

Comment:
I was wondering why this text needs to be deleted?

Reply:
This was a request from Houssein since we are creating a new codelab specifically for webfonts, he wanted that the web fonts portion be removed from the existing codelab to avoid duplication.